var/spack/repos/builtin/packages/netcdf-c/package.py

```diff
@@ -5,17 +5,6 @@
 
 from spack import *
 
```

Comment:
I think you need one more empty line here to get flake8 to pass, although I can't seem to view the Travis report.

Reply:
Thanks.  I will try that.

Reply:
Thanks.  Will try that.